src/app/application/components/route-view/RouteView.vue

```diff
@@ -198,6 +198,9 @@ watch(() => {
 }, { immediate: true })
 
 watch(() => props.name, () => {
+  if (props.name.startsWith('$routeName')) {
+    throw new Error('`$routeName` has been used for a name, please use `:name` not `name`')
+  }
```

Comment:
I bumped into a thing were I used `name="$routeName!"` instead of `:name="$routeName!"`, and it took me a good few seconds/minutes to realize.

This should prevent that from happening again, and seeing as we are already watching `props.name` it was no hassle to add.

Reply:
Following my discovery that we can't use a global `$routeName` I remove this check